### DISCUSSION TITLE
Voltage Divider on the getting started page

### DISCUSSION THREAD

**Discussion by Thewbi** [SpiceSharp/SpiceSharp#149] [General]
Hello,

first of all I am impressed by the SpiceSharp work and I also think it is great that the code is on github for everybody to look at.

I have a question about the getting started doc https://spicesharp.github.io/SpiceSharp/articles/tutorials/gettingstarted.html
I am a beginner when it comes to eletronic circuits but I do not understand why the example voltage divider using two equally valued resistors divides the input voltage by 2/3. Should it not be 1/2? I looked at https://en.wikipedia.org/wiki/Voltage_divider where Vout is given by the formula (R2 / (R1 + R2)).

Maybe I do not see why the gettings started document is right, but maybe it is worth taking a look at.

I think the image does not match the source code snippet. In the snippet the resistor values are 10k and 20k whereas on the image the resistors are 10k and 10k.

Thank you.

**Comment by svenboulanger** [comment]
Hi, I updated the schematic image a while back but did not use the correct resistor values. The bottom resistor is indeed supposed to be 20k instead of 10k.

Well spotted :-)
